osu.Game/Screens/Edit/Compose/Components/SelectionHandler.cs

```diff
@@ -219,18 +219,28 @@ internal void HandleDeselected(SelectionBlueprint blueprint)
         /// </summary>
         /// <param name="blueprint">The blueprint.</param>
         /// <param name="state">The input state at the point of selection.</param>
-        internal void HandleSelectionRequested(SelectionBlueprint blueprint, InputState state)
+        /// <returns>Whether right click was handled.</returns>
```

Comment:
This is a little specific, maybe... I believe there could be a way to make this more generic.

As this is supposed to enable aborting the selection process due to some secondary action being performed, how about inverting this, saying that the return value of `true` indicates that the selection process should be aborted, and then doing `&=` instead of `|=` in `beginClickSelection`?

Reply:
Not 100% sure I follow this, are you saying this method should be returning false when quick deletion handles?

Reply:
That's pretty much what I had in mind, yeah.

Reply:
I feel that this change reads worse (because the main purpose here is to block the input event from traversing to the context menu), but I've changed it and don't mind either way.